SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 3.x     | :white_check_mark: |
```

Comment:
Is there semantic meaning associated with a *white* check mark, as opposed to e.g. a green one?

Reply:
This is just what the GH default template creation process through the site UI used. I don't know if it is significant or not.

Reply:
<img width="277" alt="Screen Shot 2021-09-23 at 08 51 22" src="https://user-images.githubusercontent.com/1078448/134541061-17e24aa4-9fd3-49a2-9d5f-9af8938bd515.png">

Best guess is that someone just thinks the solid background for the check is more accessible/easily decipherable